Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 source 'http://gems.dev.mas.local'
 ruby '2.2.0'
 
-gem 'dough-ruby', '~> 4.0', git: 'https://github.com/moneyadviceservice/dough.git', require: 'dough'
+gem 'dough-ruby', '~> 5.0', git: 'https://github.com/moneyadviceservice/dough.git', require: 'dough'
```

Comment:
I think you should be ok to remove the `git:` part here as we're publishing the gem properly now.